<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>621</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>随意POST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>13</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>44</y>
      <width>41</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>数据:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leUrl">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>441</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>leUrl</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="leData">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>40</y>
      <width>441</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>leData</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPost">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>101</width>
      <height>57</height>
     </rect>
    </property>
    <property name="text">
     <string>POST!</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tbResult">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>601</width>
      <height>311</height>
     </rect>
    </property>
    <property name="documentTitle">
     <string/>
    </property>
    <property name="html">
     <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;tbResult&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>621</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
